src/extensions/default/BrambleUrlCodeHints/camera/selfieWidget.html

```diff
@@ -8,8 +8,29 @@
     </div>
     <div id="selfie-gallery">
         <div class="polaroid">
+        <!--<div>-->
             <img id="selfie-photo" />
         </div>
+
+    </div>
+    <div>
+      <canvas id="canvas"></canvas>
+    </div>
+    <div id="filter-list">
+        <nav>
+          <button id="resetbtn" class="btn btn-success">Reset Photo</button>
+          <button id="pinholebtn" value="pinhole" class="btn btn-primary">Pin Hole</button>
+          <button id="sepiabtn" value="sepia" class="btn btn-primary">Sepia</button>
+          <button id="contrastbtn" value="contrast" class="btn btn-primary">Contrast</button>
+        </nav>
+        <br/>
+        <nav>
+          <button id="vintagebtn" value="vintage" class="btn btn-primary">Vintage</button>
+          <button id="sunrisebtn" value="sunrise" class="btn btn-primary">Sun Rise</button>
+          <button id="glowingsunbtn" value="glowingSun" class="btn btn-primary">Glow Sun</button>
+          <button id="embossbtn" value="emboss" class="btn btn-primary">Emboss</button>
```

Comment:
These we'll somehow want to work into the main UI so you don't have to scroll.  This is something we need @flukeout to help us with, but I suspect we can put them in rows below the preview image.  We can alter how it looks, too.  It doesn't need to look so much like a polaroid and waste vertical space.  Talk to @flukeout.